lib/stack_master/parameter_resolvers/parameter_store.rb

```diff
@@ -0,0 +1,31 @@
+module StackMaster
+  module ParameterResolvers
+    class ParameterStore < Resolver
+
+      SecretNotFound = Class.new(StandardError)
```

Comment:
I think `ParameterNotFound` would be more appropriate here

Reply:
SecretNotFound was already used, but sure naming is hard :-)

Reply:
Updated in 4d9ae15